src/NATS.Client.JetStream/Models/Config.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System.Text.Json.Serialization;
+
+namespace NATS.Client.JetStream.Models;
+
+public record Config
```

Comment:
Is this StreamConfig? There are multiple configs.

Reply:
Yes, I think so, It was part of the stream creation response. Fixed now.